### PR TITLE
MON-3289: Setup merge workflow for metrics-server

### PR DESCRIPTION
**Pull request by slashpai**

### CODE REVIEW

File: .github/workflows/merge-metrics-server.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: metrics-server merger
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' #@daily
+
+jobs:
+  kube-state-metrics-merge:
```
Comment (simonpasquier) [inline .github/workflows/merge-metrics-server.yaml:9]: `metrics-server-merge` rather?

Comment (slashpai) [reply .github/workflows/merge-metrics-server.yaml:9]: aah ya typo from using ksm file :)

Comment (slashpai) [reply .github/workflows/merge-metrics-server.yaml]: fixed

**Comment by slashpai** [comment]
@simonpasquier can you review again?

**Review by simonpasquier — APPROVED** [review]

**Pull request merged**